app/src/main/res/layout/fragment_map.xml

```diff
@@ -1,13 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".fragment.navigation.MapFragment">
+    tools:context=".fragment.navigation.CourseTabFragment">
```

Comment:
Why do you have "CourseTabFragment" here ? Shouldn't it be "MapFragment" ?

Reply:
Yes indeed, a copy-pasta has gone wrong ^^